pkg/web/apps/apps_psql_repository.go

```diff
@@ -106,3 +111,19 @@ func (a *appsPostgreSQLRepository) GetAppVersionsByAppID(id string) (*[]models.V
 
 	return &versions, nil
```

Comment:
IMHO, it is another struct and in this way another object. It is returning the versions they should be in a repo for versions. It shows the versions and apps been returning in the app repository and URL. Each one should have your own files and endpoints in my point of view.

Reply:
This is a valid point but falls outside the scope of this PR, Can you raise this in chat to be discussed by a wider audience weather this rewrite is necessary. I believe that the version is never called except as part of the app so I don't believe a separation of concerns  is necessary